labelbox/schema/batch.py

```diff
@@ -121,3 +122,49 @@ def export_data_rows(self, timeout_seconds=120) -> Generator:
             logger.debug("Batch '%s' data row export, waiting for server...",
                          self.uid)
             time.sleep(sleep_time)
+
+    def delete(self) -> None:
+        """ Deletes the given batch.
+
+        Note: Batch deletion for batches that has labels is forbidden. 
+
+        Args:
+            batch (Batch): Batch to remove queued data rows from
+        """
+
+        project_id_param = "projectId"
+        batch_id_param = "batchId"
+        self.client.execute("""mutation DeleteBatchPyApi($%s: ID!, $%s: ID!) {
+            project(where: {id: $%s}) { deleteBatch(batchId: $%s) { deletedBatchId } }
+        }""" % (project_id_param, batch_id_param, project_id_param,
+                batch_id_param), {
+                    project_id_param: self.project_id,
+                    batch_id_param: self.uid
+                },
+                            experimental=True)
+
+    def delete_labels(self, labels_as_template=False) -> None:
```

Comment:
maybe better to call the flag set_labels_as_template, otherwise could be confusing what operation is being performed with them